_life/money-americans.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: post
-title: "Money and Taxes Supplement for US citizens 🗽"
+title: "Money and Taxes Supplement for US-Affiliated Persons 🗽"
```

Comment:
I know this is a long and awkward title but that's kind of the joke here please humor me

Reply:
go full Prince, "The people formerly known as Americans" :p (please don't)